EmbeddedSocialClient/sdk/src/main/java/com/microsoft/embeddedsocial/account/UserAccount.java

```diff
@@ -148,6 +158,14 @@ public void updateAccountDetails(AccountData newAccountDetails) {
 		AccountDataStorage.store(context, newAccountDetails);
 	}
 
+	/**
+	 * Updates the user's account details to reflect whether or not
+	 * the email used to authenticate the user is a device account
+	 */
+	public void updateIsDeviceAccount(boolean isDeviceAccount) {
```

Comment:
Should this be moved to AuthUtils.java?

Reply:
AccountDataStorage is currently only accessible from the account package.  This would require me to change access modifier of the class.